kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/UserTaskServiceIntegrationTest.java

```diff
@@ -234,7 +237,17 @@ public void testReleaseAndClaim() throws Exception {
             assertEquals(1, taskList.size());
             taskSummary = taskList.get(0);
             checkTaskNameAndStatus(taskSummary, "First task", Status.Reserved);
+            
+            //Try to claim task which is already reserved (Code 403)
+            taskClient.claimTask(CONTAINER_ID, taskSummary.getId(), USER_JOHN);
+
+        } catch(KieServicesHttpException e) {
+            assertEquals(Response.Status.FORBIDDEN.getStatusCode(), e.getHttpCode().intValue());
+            // expected for REST
 
+        } catch(KieServicesException e) {
+            // expected for JMS
+            
```

Comment:
@mcivantos-tribalyte you can use this method with a lambda to make newly added tests/asserts more readable https://github.com/kiegroup/droolsjbpm-integration/blob/e2129e0db207c1873823bf8b686a387e19d36e04/kie-server-parent/kie-server-tests/kie-server-integ-tests-common/src/main/java/org/kie/server/integrationtests/shared/basetests/RestJmsSharedBaseIntegrationTest.java#L154-L165

An example on how to use it is here:
https://github.com/kiegroup/droolsjbpm-integration/blob/e2129e0db207c1873823bf8b686a387e19d36e04/kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/RollingUpdateProcessServiceIntegrationTest.java#L307-L311

Reply:
+1 on using the asserClientException makes it consistent and easier to follow

Reply:
Done.